pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -821,7 +821,10 @@ func (a *ApplicationBackupController) backupResources(
 	if err = a.client.Update(context.TODO(), backup); err != nil {
 		return err
 	}
-
+	// Only on success compute the total backup size
```

Comment:
Needs to be before the call to Update() above

Reply:
Fixed.